locales/en.json

```diff
@@ -23,6 +23,8 @@
   "errors.login.bank.validBranch": "Your branch number should only contain digits",
   "errors.login.bank.validInstitution": "Your institution number should only contain digits",
   "errors.login.bank.validAccount": "Your account number should only contain digits",
+  "errors.login.taxReturn.validYear": "errors.login.taxReturn.validYear",
+  "errors.login.taxReturn.currency": "errors.login.taxReturn.currency",
```

Comment:
Can you write some best-guess error messages for this? Try to match them to the previous error messages we have.